sklearn/utils/sparsetools/setup.py

```diff
@@ -1,22 +1,15 @@
-import numpy
+# Remove in version 0.21
```

Comment:
> Apparently the lack of a setup.py and tests folder causes this test to fail..

OK I did not know that. I think this is fine like this then.

> Do we need this line in utils/setup.py?
#8854 (comment)
config.add_subpackage('sparsetools')

I think so. One way to check would be to do `python setup.py sdist` and make sure that the folder `sklearn.utils.sparsetools` is in the sdist.